res/linuxPackaging.json

```diff
@@ -18,8 +18,8 @@
     "x-scheme-handler/ftp"
   ],
   "requires": [
-    "GConf2",
-    "libXScrnSaver",
+    "(GConf2 or gconf2)",
+    "(libXScrnSaver or libXss1)",
```

Comment:
Alternatively I'll remove "Fixes" keyword so this can be merged as fix for Tumbleweed and issue will remain open for Leap users.